### DISCUSSION TITLE
Timestamps of each chunk

### DISCUSSION THREAD

**Discussion by dgoryeo** [flutydeer/audio-slicer#18] [Q&A] [answered]
Hi @flutydeer ,

Thank you for this package. Would it be possible to store / retrieve the timestamps (start - end in miliseconds) of each sliced chunk? 

Best regards

**Comment by flutydeer** [comment]
Yes, but why do you need this feature? If you want to edit markers in other applications, you can try [OpenVPI's dataset-tools](https://github.com/openvpi/dataset-tools). It can save slice markers into a csv file, which can be loaded by Audition.
![image](https://github.com/flutydeer/audio-slicer/assets/68263367/bde25284-5d89-463a-a846-f6827a4267b8)
![image](https://github.com/flutydeer/audio-slicer/assets/68263367/8565e4de-a515-40ef-a61a-d1da7c9cbfc3)

**Comment by dgoryeo** [comment]
Thanks @flutydeer . My use case is to: slice audio to chapters, transcribe each chapter as subtitles, join/stitch the subtitiles of each chapter to have complete and correct timetsamps for entire audio. I think I should be able to do that by using the .csv markers file.

**Reply by dgoryeo** [reply]
Hi @flutydeer , can the python code be used to same the chunks and the markers --i.e non-gui code.

**Reply by flutydeer** [reply]
Python code in this repo does not support markers.

**Reply by dgoryeo** [reply]
@flutydeer , I just tried the [OpenVPI's dataset-tools](https://github.com/openvpi/dataset-tools) to save audio chunks and markers. Couple of quuck questions:
- are the markers in number of frames, or miliseconds? 
- what does minimum interval mean?

Thanks

**Reply by flutydeer** [reply, marked as answer]
1. frames.
2. Please check out README in this repo.

**Reply by dgoryeo** [reply]
Thank you @flutydeer !
